zomes/blog/code/src/post_validations.rs

```diff
@@ -11,18 +11,19 @@
 // This file is WIP and placeholder to be overwritten by future
 // PR after completing the design discussion and reaching consensus
 // about these specifics.
-
-extern crate hdk;
-#[macro_use]
-extern crate serde_derive;
-extern crate serde_json;
+use serde_json;
 
 #[derive(Serialize, Deserialize)]
 struct Post {
     content: String,
     date_created: String
 }
 
+#[no_mangle]
+pub extern "C" fn validate_commit(_offset: i32) -> i32{
+    0
+}
+
 #[no_mangle]
 pub extern "C" fn validate_post(payload: String) -> bool {
     serde_json::from_str::<Post>(&payload).is_ok()
```

Comment:
Why leave this stuff in? Maybe put a comment at least indicating that validate_commit actually performs a useful function in the testing, while these other ones are staying here for discussion only purposes?

Reply:
I didn’t touch those because we another ticket that is about validation. There is a comment at the top of this file that says that this is all preliminary. But yes, I can add another comment..